zipkin-server/src/test/java/zipkin2/server/internal/ITZipkinMetricsHealth.java

```diff
@@ -60,8 +60,11 @@
   }
 
   @Test public void healthIsOK() throws Exception {
-    assertThat(get("/health").isSuccessful())
-      .isTrue();
+    Response check = get("/health");
+    assertThat(check.isSuccessful()).isTrue();
+    assertThat(check.body().string()).isEqualTo(
+      "{\"status\":\"UP\",\"zipkin\":{\"status\":\"UP\",\"details\":{\"InMemoryStorage{}\":{\"status\":\"UP\"}}}}"
```

Comment:
in preparation of directly writing the /health endpoint one day.. we should have content checks.